pkg/ccl/backupccl/restore_job.go

```diff
@@ -87,6 +88,13 @@ var useSimpleImportSpans = settings.RegisterBoolSetting(
 	false,
 )
 
+var restoreStatsInsertionConcurrency = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"bulkio.restore.insert_stats_workers",
+	"number of concurrent workers that will restore backed up table statistics",
+	5,
```

Comment:
Should we add a non-negative constraint here so that we don't have to deal with the case of it being 0 in any of the code below?

Reply:
good idea, done.

Reply:
just for clarity, I made it a PositiveInt so that the setting has to be > 0